tsconfig.json

```diff
@@ -1,14 +1,13 @@
 {
     "compilerOptions": {
-        "target": "es6",
-        "module": "commonjs",
+        "target": "ES2022",
+        "module": "Preserve",
```

Comment:
should these be upper case or lower case or does it not matter?
https://www.typescriptlang.org/tsconfig/#module

Reply:
I used the TS autocomplete suggested casing.